activestorage/test/models/attachments_test.rb

```diff
@@ -68,7 +68,7 @@ class ActiveStorage::AttachmentsTest < ActiveSupport::TestCase
   test "attach new blobs" do
     @user.highlights.attach(
       { io: StringIO.new("STUFF"), filename: "town.jpg", content_type: "image/jpg" },
-      { io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg" })
+      io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg")
```

Comment:
I see. Listening to it, I think `context_dependent` seems a good choice.

For reference, the following is the affect when `context_dependent` is enabled.

```diff
diff --git a/.rubocop.yml b/.rubocop.yml
index 2889acfdd1..cec4fcc71a 100644
--- a/.rubocop.yml
+++ b/.rubocop.yml
@@ -16,6 +16,7 @@ Style/AndOr:
 # method call.
 Style/BracesAroundHashParameters:
   Enabled: true
+  EnforcedStyle: context_dependent
```

Run `rubocop -a --only Style/BracesAroundHashParameters`.

```console
% rubocop -a --only Style/BracesAroundHashParameters
(snip)
% git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   .rubocop.yml
        modified:   actioncable/test/channel/stream_test.rb
        modified:   actionpack/test/controller/live_stream_test.rb
        modified:   actionpack/test/controller/redirect_test.rb
        modified:   actionpack/test/controller/resources_test.rb
        modified:   actionpack/test/controller/test_case_test.rb
        modified:   actionpack/test/dispatch/routing_assertions_test.rb
        modified:   actionpack/test/journey/router_test.rb
        modified:   actionview/test/actionpack/abstract/abstract_controller_test.rb
        modified:   actionview/test/template/date_helper_test.rb
        modified:   actionview/test/template/form_collections_helper_test.rb
        modified:   actionview/test/template/form_helper_test.rb
        modified:   actionview/test/template/form_options_helper_test.rb
        modified:   actionview/test/template/form_tag_helper_test.rb
        modified:   actionview/test/template/text_helper_test.rb
        modified:   activerecord/test/cases/attribute_set_test.rb
        modified:   activestorage/lib/active_storage/service/disk_service.rb
        modified:   activesupport/test/json/encoding_test.rb
        modified:   railties/lib/rails/engine/updater.rb
        modified:   railties/lib/rails/tasks/framework.rake
        modified:   railties/test/application/assets_test.rb
        modified:   railties/test/application/configuration_test.rb
        modified:   railties/test/application/mailer_previews_test.rb
        modified:   railties/test/application/middleware/cache_test.rb
        modified:   railties/test/application/middleware_test.rb
        modified:   railties/test/generators/api_app_generator_test.rb
        modified:   railties/test/generators/app_generator_test.rb
        modified:   railties/test/generators/create_migration_test.rb
        modified:   railties/test/railties/engine_test.rb
        modified:   railties/test/railties/mounted_engine_test.rb
```

It seems that there is much affect on the test codes now.